improver_tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -1282,13 +1283,19 @@ def setUp(self):
             ],
             dtype=np.float32,
         )
-        self.cube = set_up_probability_cube(
-            data, np.array([288, 290, 292], dtype=np.float32)
-        )
-        self.cube.attributes["mosg__model_configuration"] = "uk_det uk_ens"
-        self.cube.attributes[
-            "mosg__model_run"
-        ] = "uk_det:20171109T2300Z:0.500\nuk_ens:20171109T2100Z:0.500"
+        cubes = []
+        for key in ("frt", "blend_time"):
+            cube = set_up_probability_cube(
+                data,
+                np.array([288, 290, 292], dtype=np.float32),
+                **{key: dt(2017, 10, 9, 1, 0)},
```

Comment:
Maybe you mean `dt(2017, 11, 9, 1, 0)` ?
Otherwise the blend time / frt is more than a month earlier than the default validity time created by set_up_probabilty_cube, which is `(2017, 11, 10, 0, 0)`.

Reply:
Yes, that's what I meant. Now changed.